recipes/ckanweb-deploy-exts.rb

```diff
@@ -85,28 +85,28 @@
 #
 extordering =
 {
-    'xloader' => 10,
-    'datastore' => 15,
-	'validation' => 20,
-	'data_qld' => 25,
-	'data_qld_integration' => 26,
-	'data_qld_google_analytics' => 27,
-	'scheming_datasets' => 30,
-	'resource_type_validation' => 33,
-	'validation_schema_generator' => 36,
-	'dcat structured_data' => 40,
-	'qa' => 43,
-	'archiver' => 46,
-	'report' => 49,
-	'harvester_data_qld_geoscience' => 50,
-	'harvest' => 53,
-	'qgovext' => 56,
-	'ytp_comments' =>59,
-	'datarequests' => 60,
-	'csrf_filter' => 63,
-	'ssm_config' => 93,
-	'odi_certificates' => 94,
-	'resource_visibility' => 95
+	'dcat structured_data' => 5,
+	'validation' => 10,
+	'data_qld' => 15,
```

Comment:
`data_qld` should probably come first, so our site's custom theming can override anything else.

Reply:
tried that but bdd tests failed see https://github.com/qld-gov-au/ckan-qld-infrastructure/commit/671cf2f9baba22f2bc5c7b5470d88323593bcc81